packages/components/tests/dummy/app/templates/components/copy/button.hbs

```diff
@@ -0,0 +1,67 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Copy::Button Component"}}
+
+<Shw::Text::H1>Copy::Button</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Generated element</Shw::Text::H2>
+
+  <Shw::Flex as |SF|>
+    <SF.Item class="shw-component-button-generated">
+      <Shw::Label>Default</Shw::Label>
+      <p id="clipboardText">"This is the text that these buttons will copy."</p>
+      <Hds::Copy::Button @text="Copy" @clipboardText="#clipboardText" />
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H2>Content</Shw::Text::H2>
+
+  <Shw::Flex as |SF|>
+    <SF.Item @label="Default">
+      <Hds::Copy::Button @text="Copy" @clipboardText="#clipboardText" />
+    </SF.Item>
+    <SF.Item @label="Icon only">
+      <Hds::Copy::Button @isIconOnly={{true}} @text="Copy" @clipboardText="#clipboardText" />
+    </SF.Item>
+    <SF.Item {{style width="200px"}} @label="Long text">
+      <Hds::Copy::Button @text="This will throw an error because no clipboardText arg was declared." />
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H2>Sizes</Shw::Text::H2>
+
+  <Shw::Flex as |SF|>
+    {{#each @model.SIZES as |size|}}
+      <SF.Item @label={{capitalize size}}>
+        <Hds::Copy::Button @text="Copy" @size={{size}} @clipboardText="#clipboardText" />
+      </SF.Item>
+    {{/each}}
+    <SF.Item @label="Full width">
+      <Shw::Outliner {{style width="300px"}}>
+        <Hds::Copy::Button @text="Copy" @isFullWidth={{true}} @clipboardText="#clipboardText" />
+      </Shw::Outliner>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H2>States</Shw::Text::H2>
+
+  <Shw::Grid @columns={{5}} as |SG|>
+    {{#each @model.SIZES as |size|}}
+      {{#each @model.STATES as |state|}}
+        <SG.Item @label="{{capitalize size}} / {{capitalize state}}">
+          <Hds::Copy::Button
```

Comment:
I am not sure how to get the correct icon to show up on a mock state- the color is correct but the icon is not: 

![CleanShot 2023-07-12 at 12 25 36](https://github.com/hashicorp/design-system/assets/4587451/ecabdaf8-a918-4206-9d28-0de564f53307)

Reply:
The other states are all native HTML button states so I would maybe list it separately and just set the value of the `isSuccess` argument to true if that would work.

Reply:
I can't think of a simple way to do it... 🤔 (not even a complex one, tbh)

Reply:
I tried this a few different ways but was unable to make it work, so I just removed it for now.

Reply:
So probably you wouldn't want to do this since it would just be used for the showcase, but you could do this:

_components/addon/components/hds/copy/button/index.js_
```
// @tracked status = DEFAULT_STATUS;
@tracked status = this.args.status ?? DEFAULT_STATUS;
```

_components/tests/dummy/app/templates/components/copy/button.hbs_
```
<Hds::Copy::Button
    @text="Copy"
    @targetToCopy="#targetToCopy"
    @status="success"
/>
```

Reply:
I exactly didn't ship that because it would only be for the showcase 👍

Reply:
We discussed this and agreed is something I'll take care of, at the end of the process (when the component and the showcase is stable enough).